model/riscv_vmem.sail

```diff
@@ -239,10 +239,7 @@ function translate_TLB_hit forall 'v, is_sv_mode('v) . (
       match update_PTE_Bits(pte, ac) {
         None()     => TR_Address(tlb_get_ppn(sv_width, ent, vpn), ext_ptw),
         Some(pte') =>
-          if not(plat_enable_dirty_update()) then
-            // pte needs dirty/accessed update but that is not enabled
-            TR_Failure(PTW_PTE_Update(), ext_ptw)
-          else {
+          if extensionEnabled(Ext_Svadu) | not(sys_enable_svade() | sys_enable_svadu()) then {
```

Comment:
```suggestion
          if not(plat_enable_dirty_update()) | menvcfg[ADUE] == 0b0 then {
```
Shouldn't this be equivalent to the old logic and also handle the new case of "Svadu present but updates disabled"?